src/main/resources/templates/team/main.html

```diff
@@ -61,7 +61,7 @@ <h2>Týmy</h2>
                             <tr>
                                 <td th:text="${team.teamId}">8888</td>
                                 <td th:text="${team.name}">Bamboo</td>
-                                <td th:text="${team.user.username}"></td>
+                                <td th:text="${team?.user?.username}"></td>
                                 <td th:text="${team.numOfMembers}">8</td>
```

Comment:
proč jsi přidal ty otazníky jen tady a ne i k dalším (teamId, name,..)
kdyz tym neni vytvoren zadny, vykresli se "V systemu nejsou tymy",
pokud se pak vytvori novy tym, ten ma vsechny sloupecky vyplneny spravne a funguje to i bez otaznicku :)

Reply:
Hotfix pro Pavlův Hotfix. Ale jestli se ti to nezdá, napíšeme Hotfix.

Reply:
jo tak :) v pořádku

Reply:
Jako taky se mě to vela nepáčí, že je tam nekompletní kód v masteru, ale co už.